tests/output/ConsoleInteractionTest.py

```diff
@@ -1082,3 +1086,20 @@ def test_source_lines(self):
                 {abspath('file'): ('def fun():\n', '    pass  \n')})
             self.assertEqual(stdout.getvalue(),
                              "('    pass  \\n',)\n")
+
+    def test_show_bears_specified_in_args(self):
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '-B', '--bears',
+                'JavaTestBear', '--no-color')
+            self.assertEqual(retval, 0)
+            self.assertEqual(stdout.strip(), 'JavaTestBear')
+
+    def test_show_bears_specified_in_args_regex(self):
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '-B', '--bears',
+                '*Java*', '*Space*', '--no-color')
+            self.assertEqual(retval, 0)
+            for bear in stdout.strip().splitlines():
```

Comment:
This should be changed into an exact match, so the order is locked down and cant randomly change.

Probably want to compare an array of names against
`[l.strip() for l in stdout.strip().splitlines()]`

Reply:
I don't think its a good idea.
If someone wants to add a bear matching this criteria, then their changes will break this test if we do an exact match. And it can be difficult to understand the context if the person is newcomer. And this is testing what we are actually doing by running coala.

Reply:
Correct.  That is intentional.  Then we can see that new test bears have an impact on the test results.  It should require that the PR adding a test bear adds only one extra line to the output here.  Or they can choose a different name for their test bear.  Not hard.